src/mpyl/steps/deploy/k8s/chart.py

```diff
@@ -591,29 +592,41 @@ def to_deployment(self) -> V1Deployment:
         kubernetes = project.kubernetes
         defaults = self.config_defaults.resources_defaults
 
-        container = V1Container(
-            name=self.release_name,
-            image=self._get_image(),
-            env=self._get_env_vars(),
-            ports=ports,
-            image_pull_policy="Always",
-            resources=ChartBuilder._to_resource_requirements(
-                resources, defaults, self.target
-            ),
-            liveness_probe=ChartBuilder._to_probe(
+        liveness_probe = (
+            ChartBuilder._to_probe(
                 kubernetes.liveness_probe,
                 self.config_defaults.liveness_probe_defaults,
                 self.target,
             )
             if kubernetes.liveness_probe
-            else None,
-            startup_probe=ChartBuilder._to_probe(
+            else None
+        )
+
+        # add a startup probe if liveness_probe is defined
```

Comment:
Sam is going to ask for docstring documentation, hehe

Reply:
fair